views/js/runner/ui/toolbox/menu.js

```diff
@@ -43,6 +43,19 @@ define([
 ], function($, _, componentFactory, hider, stackerFactory, shortcut, namespaceHelper, menuTpl, menuItemTpl) {
     'use strict';
 
+    var _defaults = {
+        $component: $(),
+        $menuButton: $(),
+        $menuContainer: $(),
+        $menuContent: $(),
+        $menuItems: $(),
+        $menuStateIcon: $(),
+        config: {},
+        hoverIndex: null,
+        id: null,
+        menuItems: []
+    };
```

Comment:
Good catch on `_defaults` placement. I'll move that to the factory. And remove `config` as well.